draft-bbs-signatures.md

```diff
@@ -432,9 +435,9 @@ Procedure:
 
 10. B = P1 + H_s * s + H_d * domain + H_1 * msg_1 + ... + H_L * msg_L
 
-11. A = B * (1 / (SK + e))
+11. A = B * (1 / (SK + e)) // Check A or B?
 
-12. signature = (point_to_octets_min(A), e, s)
+12. signature = signature_to_octets(A, e, s)
```

Comment:
```suggestion
12. signature_octets = signature_to_octets(A, e, s)
```
Just to through an idea. Maybe use signature_octets?? That way in Verify we will have `signature = octets_to_signature(signature_octets)` which i thing is better?? This means that we will need to pass signature_octets as an input to Verify though. So both have tradeoffs.